lib/voicebase/v3/client.rb

```diff
@@ -0,0 +1,153 @@
+module VoiceBase
+  module V3
+    module Client
+      def self.extended(client, args = {})
+        client.api_host     = client.args[:host] || ENV.fetch('VOICEBASE_V3_API_HOST', 'https://apis.voicebase.com')
+        client.api_endpoint = client.args[:api_endpoint] || ENV.fetch('VOICEBASE_V3_API_ENDPOINT', '/v3')
```

Comment:
Should this come from some configurable singleton or something?

Reply:
I am just following the patterns from v1 and v2. 
https://github.com/usertesting/voicebase-client-ruby/blob/master/lib/voicebase/v1/client.rb#L11-L12
https://github.com/usertesting/voicebase-client-ruby/blob/master/lib/voicebase/v2/client.rb#L8-L9